test/Board/board.test.js

```diff
@@ -1,54 +1,53 @@
 import { Board } from '../../lib/Board/board'
+import { Token } from '../../lib/Token/token'
 
 describe('Board', () => {
     let board;
 
-    
     beforeEach(() => {
         board = new Board();
     });
 
-    describe('getCells()', () => {
-        test('will return a structure storing each individual cell in the board', () => {
-            const cellStructure = board.getCells()
-            expect(cellStructure).toBe(board.cells)
-        });    
-    });
-
-    describe('formatCells()', () => {
-        test('will format a cell without a column separator', () => {
-            const formattedCellWithSeparator = board.formatCells(board.cells[0][2], 2)
-            expect(formattedCellWithSeparator).toBe(` `)
+    describe('toString()', () => {
+        test('will display 1 row when the board is instantiated with only 1 row', () => {
+            board = new Board(1, 3);
+            const row = board.toString();
+            expect(row).toBe(`   |    |  `);
         });
 
-        test('will format a cell with a column separator', () => {
-            const formattedCellNoSeparator = board.formatCells(board.cells[0][0], 0)
-            expect(formattedCellNoSeparator).toBe(`   |`)
+        test('will display a standard, user-friendly 3x3 Tic-Tac-Toe board', () => {
+            const printedBoard = board.toString();
+            const rowOutput = `   |    |  `;
+            expect(printedBoard).toBe(`${rowOutput}\n${board.rowSeparator}\n${rowOutput}\n${board.rowSeparator}\n${rowOutput}`);
         });
     });
 
-    describe('toString()', () => {
-        test('will print 1 row when the board is instantiated with only 1 row', () => {
-            const board = new Board(1, 3)
-            const row = board.toString()
-            expect(row).toBe(`   |    |  `)
-        });
+    describe('getCells()', () => {
+        test('will return an array data structure used to store each row and column in the board', () => {
+            const cellStructure = board.getCells();
+            expect(cellStructure).toBe(board.cells);
+        });    
+    });
 
-        test('will print a standard, user-friendly 3x3 Tic-Tac-Toe board', () => {
-            const printedBoard = board.toString()
-            const rowOutput = `   |    |  `
-            expect(printedBoard).toBe(`${rowOutput}\n${board.rowSeparator}\n${rowOutput}\n${board.rowSeparator}\n${rowOutput}`)
+    describe('getRow()', () => {
+        test('will return the correlated row number for a valid position', () => {
+            const row = board.getRow(4);
+            expect(row).toBe(1);
+        });
+    });
+ 
+    describe('getColumn()', () => {
+        test('will return the correlated column number for a valid position', () => {
```

Comment:
@DeKaylaPhillips Similar question to the one for `getRow()`. Can you explain what this function is doing? Why does `board.Column(#)` return a number?

Reply:
Sure thing!

Similarly to the `getRow()` method, the `getColumn` method returns the results of the private #findRow() method which is used to retrieve the correct index number of a row in the cells array based on the position number (1-9) passed to it as an argument.

#findIndex looks through an object containing the maximum position number that would represent the final column in a row and retrieves the index number of one of the three inner array elements [[ ], [ ], [ ]] (row.rowIdx) if the given position number is either less than (which would be one of the first cells/columns*) or equal to the last cell/column in the array.

Positions 1, 2, 3 ---> [[1, 2, 3] --> Index 0 returned
Positions 4, 5, 6 ---> [4, 5, 6], --> Index 1 returned
Positions 7, 8, 9 ---> [7, 8, 9]] --> Index 2 returned

The number returned from this method is used as the first index number of the nested cells array (...this.cells[-row#-]...) when accessing a particular 'row' array element in the nested cells array.

Reply:
Sure, thing!

Similar to the `getRow()` method, the `getColumn()` method returns the results of the private #findColumn() method which is used to retrieve the correct index number of a column where a cell would live in the cells array based on the position number (1-9) passed to it as an argument.

`#findColumn` looks through an object containing row index numbers and a `column` key whose value calculates the index of a cell element at a certain given position by subtracting the given position number passed as an argument from the minimum position number that could lie within a row. 

Row 0 ---> [[1, 2, 3] --> Index 0, 1, or 2 returned
Row 1 ---> [4, 5, 6], --> Index 0, 1, or 2 returned
Row 2 ---> [7, 8, 9]] --> Index 0, 1, or 2 returned

The column index number returned from this method is used as the second index number of the nested cells array (... this.cells[-row#-][-col#-] ...) when accessing a particular cell at some column within a row array element in the nested cells array.